CODEOWNERS

```
@@ -1,5 +1,112 @@
 # Code in the `rasa.shared` package is potentially re-used by downstream dependencies
 # such as Rasa X. Hence, changes within this package require double checking.
-/rasa/shared/ @RasaHQ/enable-squad
 /docs/docs/prototype-an-assistant.mdx @ricwo @alwx
 /.github/workflows/ @RasaHQ/infrastructure
+/rasa/cli/ @RasaHQ/enable-squad
+/rasa/core/actions/ @RasaHQ/enable-squad
+/rasa/core/brokers/ @RasaHQ/enable-squad
+/rasa/core/channels/ @RasaHQ/enable-squad
+/rasa/core/featurizers/
+/rasa/core/nlg/ @RasaHQ/enable-squad
+/rasa/core/policies/ensemble.py @RasaHQ/enable-squad
+/rasa/core/policies/policy.py @RasaHQ/enable-squad
+/rasa/core/policies/registry.py @RasaHQ/enable-squad
+/rasa/core/policies/__init__.py @RasaHQ/enable-squad
+/rasa/core/policies/fallback.py @RasaHQ/research
+/rasa/core/policies/form_policy.py @RasaHQ/research
+/rasa/core/policies/mapping_policy.py @RasaHQ/research
+/rasa/core/policies/memoization.py @RasaHQ/research
+/rasa/core/policies/rule_policy.py @RasaHQ/research
+/rasa/core/policies/sklearn_policy.py @RasaHQ/research
+/rasa/core/policies/ted_policy.py @RasaHQ/research
+/rasa/core/policies/two_stage_fallback.py @RasaHQ/research
```

How about simplifying this and doing something like the following (according to the GitHub docs the **last match** takes precedence)
```suggestion
/rasa/core/policies/*.py @RasaHQ/research
/rasa/core/policies/ensemble.py @RasaHQ/enable-squad
/rasa/core/policies/registry.py @RasaHQ/enable-squad
```